### DISCUSSION TITLE
Support for Kitty images?

### DISCUSSION THREAD

**Discussion by Chickensoupwithrice** [aditya-K2/gomp#44] [Ideas]
Wondering if there could be the possibility of supporting [Kitty Icat](https://sw.kovidgoyal.net/kitty/integrations/) with uberzug as a fall back?

**Comment by akurdunkar** [comment]
gomp doesn't use Ueberzug. It uses the [ueberzug-go](https://gitlab.com/diamondburned/ueberzug-go) this is a native go library that uses child windows to display images (just like ueberzug).This enables gomp to run without any overhead on most terminal emulators

**Reply by Chickensoupwithrice** [reply]
this is cool, but it does come at a cost, namely using escape characters instead of displaying the image. im not sure if kitty would require additional dependencies

**Reply by ghost** [reply]
Also it seems that ueberzug-go doesn't support wayland.

Why kitty images anyways? Why not a cross-terminal standard like [SIXEL](https://www.arewesixelyet.com/)?
